### PR TITLE
touch_buttons: config for repeat delays

### PR DESCRIPTION
**Pull request by tpruvot**
250ms is really slow for menus (40 gcode files = 10 seconds to scroll them all)

### CODE REVIEW

**Comment by thinkyhead** [comment]
I kind of feel we should take the Jobs/Ive approach here and just decide what is best for everyone.

**Comment by tpruvot** [comment]
yep, indeed, not an easy case right now, i had to revert a part of last commits, to stay usable

**Comment by tpruvot** [comment]
and unsure about robin users, i see their sample doesnt even include touch buttons... vacations maybe

File: config/examples/Mks/Robin/Configuration.h [hunk 1]
```diff
@@ -2062,8 +2062,11 @@
 //
 // ADS7843/XPT2046 ADC Touchscreen such as ILI9341 2.8
 //
-//#define TOUCH_BUTTONS
+#define TOUCH_BUTTONS
```
Comment (thinkyhead) [inline config/examples/Mks/Robin/Configuration.h:2065]: Is the MKS Robin (a board, not a printer) guaranteed to have the touch display?

Comment (tpruvot) [reply config/examples/Mks/Robin/Configuration.h:2065]: yes the FSMC TFT & touchscreen was named MKS_ROBIN_TFT :p

image

Comment (tpruvot) [reply config/examples/Mks/Robin/Configuration.h:2065]: then i dunno, generic boards are "generic"... but well they often sell it with the touchscreen

mks brand on the tft too

**Comment by robbycandra** [comment]
I prefer 
TOUCH_EDIT_DELAY = 50
TOUCH_MENU_DELAY = 250 
for standard Marlin Configuration

File: config/examples/Mks/Robin/Configuration.h (hunk 1 above)
Comment (robbycandra) [reply config/examples/Mks/Robin/Configuration.h:2065]: Confirm, MKS Robin and MKS Robin Mini have Touch Screen installed 'cause the screen comes in one package with the board.

**Comment by tpruvot** [comment]
updated

**Pull request merged**